_data/sites.json

```
@@ -16360,6 +16360,17 @@
         ]
     },
 
+    {
+        "name": "Otto.de",
+        "url": "https://www.otto.de/myaccount/mydata",
+        "difficulty": "easy",
```

```suggestion
        "difficulty": "hard",
```
If it undergoes manual review, I think it should be hard instead

---

That's fair I wasn't quite sure if difficulty could also be easy since the user has a "delete account" button. But hard makes sense